src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestStage.java

```diff
@@ -186,6 +185,7 @@ private void putRemote(String fullFilePath, byte[] data, int retryCount)
               .setStreamingIngestClientKey(this.clientPrefix)
               .setStreamingIngestClientName(this.clientName)
               .setProxyProperties(this.proxyProperties)
+              .setDestFileName(fullFilePath)
```

Comment:
Should this only be the file name instead of full path?

Reply:
Not sure why in KC it's the file name, but I'm pretty sure we need to use fullFilePath here, and Spark Connector is using full path as well